tests/neg/i13558.check

```diff
@@ -0,0 +1,22 @@
+-- [E008] Not Found Error: tests/neg/i13558.scala:23:14 ----------------------------------------------------------------
+23 |    println(a.id) // error
+   |            ^^^^
+   |            value id is not a member of testcode.A.
+   |            An extension method was tried, but could not be fully constructed:
+   |
+   |                testcode.ExtensionA.id(a)    failed with
```

Comment:
It would be nice if we issued
```
     id(a)     failed with
```
instead, but it turns out to be devilishly complicated to do so. So I decided it's not worth going down that rabbit hole since a fix would likely make code that's already super complicated even less understandable.